File: GUI/GUI/Access.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GUI
{
   public class Access
    {
        private string employeeID, employeeName, employeeSurname, localName;
        private DateTime time;
        private bool result;

        public string EmployeeID { get => employeeID; set => employeeID = value; }
        public string EmployeeName { get => employeeName; set => employeeName = value; }
        public string EmployeeSurname { get => employeeSurname; set => employeeSurname = value; }
        public string LocalName { get => localName; set => localName = value; }
        public DateTime Time { get => time; set => time = value; }
        public bool Result { get => result; set => result = value; }

        public Access() { }

        public Access(string employeeID, string employeeName, string employeeSurname, string roomName, DateTime time, bool result)
        {
            EmployeeID = employeeID;
            EmployeeName = employeeName;
            EmployeeSurname = employeeSurname;
            LocalName = roomName;
            Time = time;
            Result = result;
        }
    }
}

File: RestfulService/src/rest/Access.java
package rest;
import java.sql.Timestamp;



/**
 * Class that represents an access in a local or a attempted one 
 *
 */
public class Access {
	private String employeeId,employeeName, employeeSurname, localName;
	private Timestamp time;
	private Boolean result;
	
	public Access() {}

	public Access(String employeeId, String employeeName, String employeeSurname, String localName, Timestamp time,
			Boolean result) {
		this.employeeId = employeeId;
		this.employeeName = employeeName;
		this.employeeSurname = employeeSurname;
		this.localName = localName;
		this.time = time;
		this.result = result;
	}
	public String getEmployeeId() {
		return employeeId;
	}
	public void setEmployeeId(String employeeId) {
		this.employeeId = employeeId;
	}
	public String getEmployeeName() {
		return employeeName;
	}
	public void setEmployeeName(String employeeName) {
		this.employeeName = employeeName;
	}
	public String getEmployeeSurname() {
		return employeeSurname;
	}
	public void setEmployeeSurname(String employeeSurname) {
		this.employeeSurname = employeeSurname;
	}
	public String getLocalName() {
		return localName;
	}
	public void setLocalName(String localName) {
		this.localName = localName;
	}
	public Timestamp getTime() {
		return time;
	}
	public void setTime(Timestamp time) {
		this.time = time;
	}
	public Boolean getResult() {
		return result;
	}
	public void setResult(Boolean result) {
		this.result = result;
	}

}
